agent/local/state.go

```diff
@@ -0,0 +1,972 @@
+package local
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/consul/acl"
+	"github.com/hashicorp/consul/agent/structs"
+	"github.com/hashicorp/consul/agent/token"
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/lib"
+	"github.com/hashicorp/consul/types"
+)
+
+// permissionDenied is returned when an ACL based rejection happens.
```

Comment:
This can be deleted - the `acl` package has the standard form of this (and I don't think this is referenced).

Reply:
done